test/libsolidity/SemanticTest.cpp

```diff
@@ -31,6 +31,7 @@
 #include <optional>
 #include <stdexcept>
 #include <utility>
+#include <functional>
```

Comment:
This list is sorted alphabetically so far, can you move this one up?

Reply:
Done.